neo4j/driver_with_context.go

```diff
@@ -452,6 +452,10 @@ func ExecuteQuery[T any](
 	newResultTransformer func() ResultTransformer[T],
 	settings ...ExecuteQueryConfigurationOption) (res T, err error) {
 
+	if driver == nil {
+		return *new(T), errors.New("driver cannot be nil")
```

Comment:
is there any specific error type for argument errors? 


And why not return `null, error` instead of `new T, error`?

Reply:
If `T` is not a pointer type, you cannot return `nil`.

Reply:
Fixed in 25ab716703795f2888867bb719b8d380e3b8a408